#!/bin/sh -e
## 60-kolabdcachetool-cachefiles.dpatch by Noèl Köthe <noel@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: This is the patch to correct the path of the cachefiles

if [ $# -lt 1 ]; then
echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
      -patch) patch $patch_opts -p1 < $0;;
      -unpatch) patch $patch_opts -p1 -R < $0;;
      *)
      echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
      exit 1;;
esac

exit 0

@DPATCH@
--- kolabd-1.9.4/kolabdcachetool.orig	2006-01-26 13:06:41.000000000 +0100
+++ kolabd-1.9.4/kolabdcachetool	2006-01-26 13:07:22.000000000 +0100
@@ -25,13 +25,13 @@
 usage if ($func !~ /list/i && $func !~ /del/i && $func !~ /flush/i);
 
 if ($cache =~ /mbox/i) {
-    dbmopen(%db, '/kolab/var/kolab/mailbox-uidcache.db', 0666)
+    dbmopen(%db, '/var/lib/kolab/mailbox-uidcache.db', 0666)
         || die "Unable to open mail uid cache";
 } elsif ($cache =~ /gyard/i) {
-    dbmopen(%db, '/kolab/var/kolab/graveyard-uidcache.db', 0666)
+    dbmopen(%db, '/var/lib/kolab/graveyard-uidcache.db', 0666)
         || die "Unable to open graveyard uid cache";
 
-    dbmopen(%db2, '/kolab/var/kolab/graveyard-tscache.db', 0666)
+    dbmopen(%db2, '/var/lib/kolab/graveyard-tscache.db', 0666)
         || die "Unable to open graveyard timestamp cache";
 } else { usage; }
 
